public/css/index.css

```diff
@@ -1,3131 +1 @@
-.emoji {
```

Comment:
Why deleted `public/css/index.css` ? I don't think is a good idea to delete this.

Reply:
It's not deleted it is minimized